### ISSUE TITLE
Test loss diverges during training nyud-fcn32s-color

### ISSUE THREAD

**Issue by howard-mahe** [MarvinTeichmann/tensorflow-fcn#40]
Hello,

After successfully train fcn8s-atonce on my own dataset with _Caffe_, I wanted to familiarize with your implementation of FCN in _Tensorflow_. I decided to start with the training of **nyud-fcn32s-color** on NYUDv2 dataset (40 classes challenge) with **heavy learning** strategy (batch size: 1, unnormalized loss, lr: 1e-10, momentum: 0.99)

I forked your tensorflow-fcn repo. Here's mine: https://github.com/howard-mahe/tensorflow-fcn
I've made some simple modifications in order to:
- create NYUDv2DataHandler.py, equivalent of nyud_layers.py in _Caffe_
- modifly loss.py to use an unnormalized loss
- implement a training script 

Training goes well in the first iterations but quickly my test loss starts to diverge but the most surprising is that my test metrics (global accuracy, mean accuracy per class, mean IoU) doesn't collapse at all, but oscillate a lot.

![logs](https://user-images.githubusercontent.com/17425914/28676016-ccfcd0f2-72e9-11e7-8d91-4302982c46cf.png)

Regarding FCN paper, I would expect the following results:
- gacc:  61.8%
- macc: 44.7%
- mIoU: 31.6%

Can anyone have a look to my repo and let me if I made something wrong ?
This issue drove me crazy for a lot of time now.

Thanks a lot for any feedbacks.
